x/supplier/keeper/msg_server_create_claim_test.go

```diff
@@ -0,0 +1,125 @@
+package keeper_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	keepertest "github.com/pokt-network/poktroll/testutil/keeper"
+	"github.com/pokt-network/poktroll/testutil/sample"
+	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
+	"github.com/pokt-network/poktroll/x/supplier/keeper"
+	"github.com/pokt-network/poktroll/x/supplier/types"
+	suppliertypes "github.com/pokt-network/poktroll/x/supplier/types"
+)
+
+func TestMsgServer_CreateClaim_Success(t *testing.T) {
+	appAddr, supplierAddr := sample.AccAddress(), sample.AccAddress()
+
+	// TODO_IN_THIS_COMMIT: dedup & refactor to a test helper.
+	sessionMockMap := keepertest.SessionMetaFixturesByAppAddr{
```

Comment:
Per one of my other comments. Reconsider: s/mock/fixture or the use of this structure at all.